chatbot-core/frontend/src/app/admin/assistants/PersonaTable.tsx

```diff
@@ -126,7 +126,7 @@ export function PersonasTable() {
                     className="mr-1 my-auto cursor-pointer"
                     onClick={() =>
                       router.push(
-                        `/admin/assistants/${persona.id}?u=${Date.now()}`
+                        `/admin/assistants/?id=${persona.id}?u=${Date.now()}`
```

Comment:
check this logic, why are there two `?` in search params

Reply:
i forgot
fixed